src/System.Private.CoreLib/shared/System/Buffers/Text/Utf8Formatter/FormattingHelpers.cs

```diff
@@ -242,5 +242,15 @@ public static uint DivMod(uint numerator, uint denominator, out uint modulo)
         }
 
         #endregion Math Helper methods
+
+        //
+        // Enable use of ThrowHelper from TryFormat() routines without introducing dozens of non-code-coveraged "bytesWritten = 0; return false" boilerplate.
+        //
+        public static bool TryFormatThrowFormatException(out int bytesWritten)
+        {
+            bytesWritten = 0;
```

Comment:
Nit: swap this line with the line immediately below. Otherwise the write has an observable side effect that the JIT cannot elide.
(Don't let this block checkin.)